invertedai_cpp/invertedai/session.cc

```diff
@@ -172,9 +183,20 @@ const std::string Session::request(
       is.push(src);
       std::stringstream strstream;
       boost::iostreams::copy(is, strstream);
-      return strstream.str();
+
+      const std::string output = strstream.str();
+      if (this->is_logging){
+        this->logger.append_request(output,mode);
```

Comment:
should this be `append_response` ?
```suggestion
        this->logger.append_response(output,mode);
```

Reply:
Yes good catch.